x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/api.ts

```diff
@@ -25,8 +25,8 @@ import {
   UpdateAlertStatusProps,
   CasesFromAlertsResponse,
 } from './types';
-import { resolvePathVariables } from '../../../../management/pages/trusted_apps/service/utils';
 import { isolateHost, unIsolateHost } from '../../../../common/lib/host_isolation';
+import { resolvePathVariables } from '../../../../common/utils/resolve_path_variables';
```

Comment:
so this one I already have done in https://github.com/elastic/kibana/pull/99795/commits/46c187d66aea1cfaeb1d48efb179167737b4c0b0

Reply:
oh gotcha! thanksss thankss

Reply:
Hi @ashokaditya this PR actually promotes this utility to be higher in the chain to `security_solution/public/common/**`. It will be reused by @parkiino as well for the stuff she is doing under the detections section of the code.

Reply:
Ah I see. 👍